app/models/marc_indexer.rb

```diff
@@ -0,0 +1,161 @@
+$:.unshift './config'
+class MarcIndexer < Blacklight::Marc::Indexer
```

Comment:
I don't think we're going to switch to traject any time soon. This and those `config/` additions should probably be removed.